starter-code/src/App.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import "../public/styles/App.css";
+import NavBar from "../components/NavBar";
+import Titles from "../components/Titles";
+import Awesome from "../components/Awesome";
+import LowDiv from "../components/LowDiv"
+
+const App = () => {
+    return (
+        <div>
+            <div className="main-div">
+                <NavBar></NavBar>
+                <Titles></Titles>
+                <Awesome></Awesome>
```

Comment:
Since you're not using props.children in these components, you could call them with a self closing tag.

```
  <NavBar />
        <Titles />
        <Awesome />
      </div>
      <LowDiv/>
```